projects/go-lib/src/lib/components/go-timepicker/go-timepicker.component.scss

```diff
@@ -0,0 +1,13 @@
+.go-timepicker {
+  position: relative;
+}
+
+.go-timepicker__toggle {
+  position: absolute;
+  right: 0.25rem;
+  top: 0.25rem;
+}
+
+.go-form__input{
+  cursor: pointer;
+}
```

Comment:
We shouldn't be adding this to every input. Let's add a class to the input and add this css to that so that we're only affecting what we need to